src/middlewared/middlewared/plugins/system.py

```diff
@@ -1359,11 +1381,59 @@ def set_crash_reporting(self):
         CrashReporting.enabled_in_settings = self.middleware.call_sync('system.general.config')['crash_reporting']
 
 
+async def _update_birthday_data(middleware, birthday=None):
+    middleware.logger.debug('Synchornization _update_birthday_data interface')
+    # Check if it is exists already
+    system_obj = (await middleware.call('system.info'))
```

Comment:
Please dont use superfluous parenthesis, we dont this anywhere else in our code
just  `await middleware.call('system.info')` is OK